scripts/workers/code_upload_submission_worker.py

```diff
@@ -127,14 +236,104 @@ def create_job_object(message, environment_image):
     # Create the specification of deployment
     spec = client.V1JobSpec(backoff_limit=1, template=template)
     # Instantiate the job object
-    job = client.V1Job(
-        api_version="batch/v1",
-        kind="Job",
-        metadata=client.V1ObjectMeta(
-            name="submission-{0}".format(submission_pk)
+    job = get_job_object(submission_pk, spec)
+    return job
+
+
+def create_static_code_upload_submission_job_object(message):
+    """Function to create the static code upload pod AWS EKS Job object
+
+    Arguments:
+        message {[dict]} -- Submission message from AWS SQS queue
+
+    Returns:
+        [AWS EKS Job class object] -- AWS EKS Job class object
+    """
+
+    PYTHONUNBUFFERED_ENV = client.V1EnvVar(name="PYTHONUNBUFFERED", value="1")
+    # Used to create submission file by phase_pk and selecting dataset location
+    MESSAGE_BODY_ENV = client.V1EnvVar(name="BODY", value=json.dumps(message))
+    submission_pk = message["submission_pk"]
+    challenge_pk = message["challenge_pk"]
+    phase_pk = message["phase_pk"]
+    image = message["submitted_image_uri"]
+    # Get init container
+    init_container = get_init_container(submission_pk)
+    # Get dataset volume and volume mounts
+    volume_mount_list = get_volume_mount_list("/dataset", True)
+    volume_list = get_volume_list()
+    # Create and Mount Script Volume
+    script_config_map_name = "script-config"
+    create_script_config_map(script_config_map_name)
+    script_volume_name = "scripts-dir"
+    script_volume = get_config_map_volume_object(
+        script_config_map_name, script_volume_name
+    )
+    volume_list.append(script_volume)
+    script_volume_mount = get_volume_mount_object(
+        "/scripts", script_volume_name, True
+    )
+    volume_mount_list.append(script_volume_mount)
+    # Create and Mount submission Volume
+    submission_path = "/submission"
+    SUBMISSION_PATH_ENV = client.V1EnvVar(
+        name="SUBMISSION_PATH", value=submission_path
+    )
+    submission_volume_name = "submissions-dir"
+    submission_volume = get_empty_volume_object(submission_volume_name)
+    volume_list.append(submission_volume)
+    submission_volume_mount = get_volume_mount_object(
+        submission_path, submission_volume_name
+    )
+    volume_mount_list.append(submission_volume_mount)
+    # Pre-Stop Container Hook to Submit file
+    submission_curl_request = get_static_code_upload_submission_file_curl(
+        challenge_pk, phase_pk, submission_pk, submission_path
+    )
+    submission_file_not_present_curl_request = (
+        get_static_code_upload_no_submission_file_curl(
+            challenge_pk, phase_pk, submission_pk
+        )
+    )
+    pre_stop_exec_command = (
+        'sh /scripts/make_submission.sh {} "{}" "{}"'.format(
```

Comment:
Here you need to execute `sh /scripts/make_submission.sh` and all required parameters needed for `make_submission.sh` should be environment variables